### PR TITLE
WW-4069 upgrade to DWR 3 with a showcase

### PR DESCRIPTION
**Pull request by yasserzamani**

### CODE REVIEW

File: plugins/dwr/src/main/java/org/apache/struts2/validators/DWRValidator.java [hunk 1]
```diff
@@ -32,7 +32,7 @@
 import org.apache.struts2.dispatcher.RequestMap;
 import org.apache.struts2.dispatcher.SessionMap;
 
-import uk.ltd.getahead.dwr.WebContextFactory;
+import org.directwebremoting.WebContextFactory;
```
Comment (lukaszlenart) [inline plugins/dwr/src/main/java/org/apache/struts2/validators/DWRValidator.java]: It was that easy??? wow :)

File: pom.xml [hunk 2]
```diff
@@ -650,7 +650,7 @@
             <dependency>
                 <groupId>org.directwebremoting</groupId>
                 <artifactId>dwr</artifactId>
-                <version>2.0.11-RELEASE</version>
+                <version>3.0.2-RELEASE</version>
```
Comment (lukaszlenart) [inline pom.xml]: 👏

File: plugins/dwr/src/main/java/org/apache/struts2/validators/DWRValidator.java (hunk 1 above)
Comment (yasserzamani) [reply plugins/dwr/src/main/java/org/apache/struts2/validators/DWRValidator.java]: Fortunately yes, I enjoyed it too :)

**Comment by lukaszlenart** [comment]
LGTM 👍

**Pull request merged**